assets/javascripts/discourse/components/discourse-reactions-actions-button.gjs

```diff
@@ -0,0 +1,11 @@
+import { hash } from "@ember/helper";
+import MountWidget from "discourse/components/mount-widget";
+
+const ReactionsActionButton = <template>
+  <MountWidget
+    @widget="discourse-reactions-actions"
+    @args={{hash post=@post}}
```

Comment:
We'll need to add the comment so that ember-template-lint ignores this

(it's not worth us spending the time to update `<MountWidget`, since it'll hopefully be removed in the next few months)

Reply:
I added the comments